visual-testing-app/webpack.config.js

```diff
@@ -22,18 +22,40 @@ if (!info) {
 }
 
 module.exports = {
-  mode: 'development',
+  target: 'web',
+  mode: 'production',
   stats: 'minimal',
   entry: './visual-testing-app/src/index.js',
   output: {
-    filename: 'main.js',
+    filename: '[name].js',
     path: path.resolve(__dirname, 'dist'),
+    publicPath: '/',
   },
+  optimization: {
+    minimize: false,
+    splitChunks: {
+      cacheGroups: {
+        vendors: {
+          test: /[\\/]node_modules[\\/]/,
+          name: 'vendors',
+          chunks: 'all',
+          priority: -20,
+        },
+        'ui-kit': {
+          test: /ui-kit.esm/,
+          name: 'ui-kit',
+          chunks: 'all',
+          priority: -15,
```

Comment:
Funky numbers?

Reply:
copy paste from shopify 😆

Reply:
Haha. Maybe add a comment then 😂

Reply:
The numbers are similar in the webpack docs. The best place to learn about splitChunks is [there](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunks-cachegroups-priority) anyways 🤷‍♂️